includes/CVUniverse.cxx

```diff
@@ -849,8 +851,8 @@ double CVUniverse::GetWeight() const {
   wgt_rpa = GetRPAWeight();
 
   // MINOS efficiency
-  if (!m_is_truth && GetBool("isMinosMatchTrack"))
-    wgt_mueff = GetMinosEfficiencyWeight();
+//  if (!m_is_truth && GetBool("isMinosMatchTrack"))
```

Comment:
We obviously don't want to remove these weights on the master branch. This was just done for the closure.

@egranadosvazquez do you remember if we must remove these lines for the closure test? Or instead did we just remove them to be safe?

Reply:
These weights or any of the others.